migration/clean-up-scripts/vault_dedupe_helper.py

```diff
@@ -0,0 +1,74 @@
+###
+# A script that will produce a csv-like report to assist with de-duplication. 
+# Outputs vaultName, vaultUUID, itemCount, vault creation date, vault updated, number of users
+###
+import os
+import subprocess
+import csv
+import json
+from datetime import datetime
+
+scriptPath = os.path.dirname(__file__)
+outputPath = scriptPath  # Optionally choose an alternative output path here.
+
+# Get a list of vaults the logged-in user has access to
+def getVaults():
+    try:
+        return subprocess.run("op vault list --group=Owners --format=json", shell=True, check=True, capture_output=True).stdout
```

Comment:
Since the assumption is that this will be run by an Owner (because it needs to be run by someone with access to all shared vaults), the use of `op vault list --group` is required to ensure that all shared vaults are returned.

`op vault list` will not return vaults an owner only has "manage vault" permissions for, and thus won't produce a comprehensive list of shared vaults since I am assuming that Owners will not have "view items" or greater permissions on all shared vaults.

Reply:
I was mistakenly assuming that this script would output a list of vaults that have a name appearing more than once, filtered from a list of all vaults that the authed user/service account has access to. I understand the purpose more clearly now (just get _all_ shared vaults in the account).

Reply:
That's an important insight, though, Pike. 
* Do you think most people would have the same expectation as you? In other words am I violating a likely-common sense assumption that I need to rethink?
* Regardless of the above, do you think that proper documentation would help? 
* Or should the script behave differently?
* Or should the user be given an option to run the script one way or the other (this is my least favourite approach).